src/server/controllers/users.controller.js

```diff
@@ -598,6 +607,70 @@ export function getMentoringRequestStatus(req, res, next) {
     });
 }
 
+export function activeBookmark(req, res, next) {
+  User.findOne({ _id: req.user._id }).exec()
+    .then((user) => {
+      user.bookmark.push(req.body.id);
+      user.save();
```

Comment:
But I need at least once finding because I need to find my bookmarked list and then update it.
After that, I don't need to use update method, cause I already found me and just saving is simple.
But return, is good idea. Even not used now, for later's use. Applied!